packages/devtools_app/lib/src/logging/html_logging_screen.dart

```diff
@@ -70,6 +75,27 @@ class HtmlLoggingScreen extends HtmlScreen {
       true,
     );
 
+    final _search = (String text) {
+      controller.search(text);
+    };
+
+    final textFiled = CoreElement('input', classes: 'form-control input-sm')
+      ..setAttribute('type', 'text')
+      ..setAttribute('style', 'width:480px;')
+      ..setAttribute('placeholder', 'input text')
+      ..id = 'uri-field';
```

Comment:
why is the id uri-field. That seems unrelated and generally we should not be adding ids.